app/components/search_input_component/search_input_component_controller.js

```diff
@@ -149,18 +150,22 @@ export default class extends Controller {
 
     #renderLine(item) {
         let template = this.templateTarget.content
-        let newElement = template.firstElementChild
-        let string = newElement.outerHTML
-
+        let newElement = template.firstElementChild.outerHTML
         Object.entries(item).forEach( ([key, value]) => {
             key = key.toString().toUpperCase()
             if (key === 'TYPE'){
                 value  = value.toString().split('/').slice(-1)
             }
+            if (key === 'ACRONYM'){
```

Comment:
specif code of Agents in generic stimulus controller

Reply:
I couldn't find a better solution